internal/pkg/service/buffer/api/service/mapper/receiver_test.go

```diff
@@ -1,4 +1,4 @@
-package mapper
+package mapper_test
```

Comment:
Testy su iba pre `receiver` mapper, co je vlastne OK, lebo to testuju E2E testy, ale ked tu tie testy su, tak som ich nechal.